CourseStream/CourseStream.Data/EnrolledStudents.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace CourseStream.Data;
+
+public class EnrolledStudent
```

Comment:
Probably it makes sense to rename this to Enrollment. Not a requirement, just a nit

Reply:
You have to store the enrollment timestamp

Reply:
Added timestamp